lib/news_scraper/trainer/preset.rb

```diff
@@ -0,0 +1,76 @@
+module NewsScraper
+  module Trainer
+    class Preset
```

Comment:
I would rather this be called `PresetX` instead of `Preset` since it implies that it's a class for presets, which it's not really.